src/scripts/get_cicero_cmds.pl

```diff
@@ -9,21 +9,22 @@
 use Pod::Usage;
 use Data::Dumper;
 use File::Basename;
-use TdtConfig; 
+use TdtConfig;
 
 if (@ARGV == 0){
-	pod2usage(1); 
-	exit 1; 
+	pod2usage(1);
+	exit 1;
 }
 
 my $out_prefix;
-my $queue = $ENV{"AFC_DEFAULT_QUEUE"}; 
+my $queue = $ENV{"AFC_DEFAULT_QUEUE"};
 
 my ( $help, $man, $version, $usage );
 my $bam_file;
-my $genome; 
+my $genome;
 my $read_length = 100;
 my $output_dir;
+my $cluster_arg = 10;
```

Comment:
This is in master already, please rebase the branch to pull in those changes that will make reviewing this much more straightforward.